index.html

```diff
@@ -426,6 +426,39 @@
 
 </section>
 
+<section class="introductory">
+
+## Audience for this Document
+
+The primary audiences for this document are
+
+ * browser developers
+
+ * authors of W3C specifications
+
+ * reviewers of W3C specifications
```

Comment:
Maybe we should be clear that it's applicable to all web specifications, not only ones developed at W3C.